src/mainboard/clevo/tgl-u/ramstage.c

```diff
@@ -112,11 +112,41 @@ static void set_fan_curve(void)
 	}
 }
 
+static void set_wifi_bt_radios(void)
+{
+    bool enable = true;
+
+#if CONFIG(DRIVERS_EFI_VARIABLE_STORE)
+    struct region_device rdev;
+    enum cb_err ret;
+
+    uint32_t size = sizeof(enable);
+
+    const EFI_GUID dasharo_system_features_guid = {
+		0xd15b327e, 0xff2d, 0x4fc1, { 0xab, 0xf6, 0xc1, 0x2b, 0xd0, 0x8c, 0x13, 0x59 }
+	};
+
+    if (smmstore_lookup_region(&rdev))
+		goto efi_err;
+
+    ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "EnableWifiBt",
```

Comment:
Indentation is still not correct here

Reply:
fixed in 32991f56e6156a4c9daeb11f5e8be5882ac31396